vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/WaitSteps.java

```diff
@@ -394,6 +394,18 @@ public void waitForScroll()
         javascriptActions.waitUntilScrollFinished();
     }
 
+    /**
+     * Waits <b>period</b> for debug purposes; Always fails by using
+     * @param period Total waiting time according to
+     *               <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO 8601</a> standard
+     */
+    @When("I wait `$period` for debug")
+    public void waitForDebug(Duration period)
```

Comment:
I don't think plugin-web-app is a correct place for the step, it could be used outside web context